export.js

```diff
@@ -0,0 +1,61 @@
+const files = require("./lib/files")
+const stringify = require("json-stringify-pretty-compact")
+const XLSX = require('xlsx');
+const cmdLine = require('./lib/command-line-parse')
+
+const log = (out) => console.log(out)
+const sheet2Json = (worksheet) => stringify(XLSX.utils.sheet_to_json(worksheet), { maxLength: 420, indent: 2 })
+const loadWorkbook = (filename) => XLSX.readFile(filename)

```

Comment:
I do not think we need that. A `XLSX.readFile(filename)` is descriptive enough. Use `XLSX.readFile(filename)` directly.